setup.py

```diff
@@ -64,10 +64,12 @@
 install_requires = [
     'py2neo==2.0.8',
     'simplejson>=3.8.2',
+    'Flask>=0.11.1',
+    'Flask-CORS>=2.1.0',
     'Flask-BabelEx>=0.9.2',
     'Flask-Cache>=0.13.1',
     'Flask-KVSession>=0.6.2',
-    'Flask>=0.11.1',
+    'invenio[minimal]>=3.0.0a2,<3.1.0',
```

Comment:
I would avoid using `invenio` package as it might cause problems with package versions you define here.